Quran/SettingsViewController.swift

```diff
@@ -36,6 +36,7 @@ class SettingsViewController: BaseTableBasedViewController {
             guard let `self` = self else { return }
             let item = ds.item(at: indexPath)
             item.onSelection(self)
+            self.tableView.deselectRow(at: indexPath, animated: true)
```

Comment:
I have added this line as when app navigates back from Appstore, the cell remains highlighted. 
To remove highlighted state, I added this line.

Reply:
That looks perfect, thank you.